Felienne/hedy
# [FEATURE] Adds blur option to code editor

**Description**
In this PR we add a nice QoL feature that enables users to blur their code. As suggested this is a nice help when wanting to show your program to someone without already spoiling what is going to happen because the code and output window are next to each other. We use some manual styling to fix this, by adding/removing the `blur` styling attribute to the editor element on click of the corresponding icon. We can simply change the "amount of blur" by editing the value in `app.ts`, which is currently set at `3px`.

**Screenshots**
![Schermafbeelding 2022-07-08 om 08 38 33](https://user-images.githubusercontent.com/38155424/177932421-b16631e8-4090-4a57-9e5c-b6f2576f29ad.png)
![Schermafbeelding 2022-07-08 om 08 38 39](https://user-images.githubusercontent.com/38155424/177932426-b1e28b86-2018-403b-9642-7f5f9963f608.png)


**Fixes**
This PR fixes #3013.

**How to test**
Notice the new icon in the bottom-right corner of the code editor. When clicked the icon should change and the code editor should become blurred. Pressing again should reverse this step.
